rudder-webapp/SOURCES/Makefile

```diff
@@ -34,6 +34,7 @@ MAVEN_PROXY := $(if $(PROXY), $(shell echo "$(PROXY)" |  perl -ne ' \
 .DEFAULT_GOAL := localdepends
 
 RUDDER_VERSION_TO_PACKAGE = <put Rudder version or version-snapshot here>
+RUDDER_MAJOR_VERSION := $(shell echo ${RUDDER_VERSION_TO_PACKAGE} | cut -d'.' -f 1-2)
```

Comment:
When downloading the doc: `http://www.rudder-project.org/doc-${RUDDER_MAJOR_VERSION}/`